queries.sql

```diff
@@ -20,3 +20,40 @@ SELECT * from animals WHERE neutered = true;
 SELECT * FROM animals WHERE name NOT LIKE 'Gabumon';
 
 SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg<=17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS null;
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
```

Comment:
- [ ] Here inside a transaction delete all records in the animals table, then roll back the transaction.
```sql
   BEGIN;

   DELETE FROM animals WHERE ;
   SELECT COUNT(*) FROM ANIMALS;

   ROLLBACK;
   SELECT COUNT(*) FROM ANIMALS;
```

Reply:
But the output gave same result? Right? Why am I penalized for coding style?

Reply:
Here the point where you might be missing it @Shedrack-Sunday  is that @Deepakdanger  is following through with the project requirements where you are supposed to verify that the action being done has effected or not before and after a rollback. 

But since the queries screenshots show that you returned the correct output it is all okay.

Reply:
Based on project requirements, the confirmation comes after the ROLLBACK. Thats why I was asking for the reason I was not approved.

![IMG_20221207_163306](https://user-images.githubusercontent.com/50990925/206222065-7234baf7-0f5d-4444-bd89-b72df12710b6.jpg)